### PR TITLE
Feature/response annotations

### LINKED ISSUE #1397: Incident response - #3 response annotations
Part of #1282 

Something should annotate the incident citation page to indicate it has an incident response. Some ideas include:

- the timeline can have a special symbol for the response.
- The header of the citation page can indicate there is an incident response with a badge.

### PR DESCRIPTION
**Pull request by clari182**
- resolves #1397

Responded incident: https://deploy-preview-1405--staging-aiid.netlify.app/cite/51

### **- Badge that indicates the incident has been `Responded`** 
image

### **- Green event on the timeline indicating that it's a response**
image

### CODE REVIEW

File: site/gatsby-site/src/components/visualizations/Timeline.js [hunk 1]
```diff
@@ -93,23 +91,50 @@ const DataPoint = ({ bucket, groupRadius, radius, yScale }) => {
...
+            {bucket[0].isResponse && (
+              <>
+                {' '}
+                - <Trans>Response</Trans>
+              </>
+            )}
           </text>
         </a>
       )}
+      {bucket[0].isResponse && <Badge color={'success'}>RESPONSE_TAG</Badge>}
```
Comment (pdcp1) [inline site/gatsby-site/src/components/visualizations/Timeline.js:137]: @clari182 Why do we have the text `RESPONSE_TAG` here?
image

**Review by pdcp1 — CHANGES_REQUESTED** [review]
We should add tests to both new features (top Badge and timeline)

File: site/gatsby-site/src/components/visualizations/Timeline.js (hunk 1 above)
Comment (clari182) [reply site/gatsby-site/src/components/visualizations/Timeline.js]: Removed!

**Review by pdcp1 — CHANGES_REQUESTED** [review]
We are good with the badge deletion on the timeline. But we still need to add tests to these two new UI changes.

File: site/gatsby-site/cypress/e2e/cite.cy.js [hunk 2]
```diff
@@ -401,4 +401,12 @@ describe('Cite pages', () => {
...
     );
   });
+
+  it('Should display response in timeline and as badge', () => {
+    cy.visit('/cite/51#r1765');
+
+    cy.get('[data-cy="responded-badge"]').should('exist');
+
+    cy.get('[data-cy="timeline-text-response"]').should('exist');
+  });
```
Comment (pdcp1) [inline site/gatsby-site/cypress/e2e/cite.cy.js:411]: This test looks good. In addition, we should have another test for an incident without responses and check that the Response badge and the green text in Timeline are not displayed in order to cover both scenarios.

**Review by pdcp1 — CHANGES_REQUESTED** [review]

**Review by pdcp1 — APPROVED** [review]

**Pull request merged**